plugins/GSdx/GSState.cpp

```diff
@@ -1817,30 +1876,32 @@ void GSState::Read(uint8* mem, int len)
 	GSVector4i r(sx, sy, sx + w, sy + h);
 
 	// Function is called from the EE thread. Unforunately gl stuff can only be used from a single thread (AKA MTGS)
-	if (GLLoader::in_replayer) {
+	if (GLLoader::in_replayer)
+	{
 		GL_CACHE("Read! len=%d SBP=%05x SBW=%d SPSM=%s SSAX=%d SSAY=%d RRW=%d RRH=%d",
-				len, (int)m_env.BITBLTBUF.SBP, (int)m_env.BITBLTBUF.SBW, psm_str(m_env.BITBLTBUF.SPSM), sx, sy, w, h);
+				 len, (int)m_env.BITBLTBUF.SBP, (int)m_env.BITBLTBUF.SBW, psm_str(m_env.BITBLTBUF.SPSM), sx, sy, w, h);
 	}
 
-	if(!m_tr.Update(w, h, GSLocalMemory::m_psm[m_env.BITBLTBUF.SPSM].trbpp, len))
+	if (!m_tr.Update(w, h, GSLocalMemory::m_psm[m_env.BITBLTBUF.SPSM].trbpp, len))
 	{
 		return;
 	}
 
-	if(!m_init_read_fifo_supported)
+	if (!m_init_read_fifo_supported)
 	{
-		if(m_tr.x == sx && m_tr.y == sy)
+		if (m_tr.x == sx && m_tr.y == sy)
 		{
 			InvalidateLocalMem(m_env.BITBLTBUF, r);
 		}
 	}
 
 	m_mem.ReadImageX(m_tr.x, m_tr.y, mem, len, m_env.BITBLTBUF, m_env.TRXPOS, m_env.TRXREG);
 
-	if(s_dump && s_save && s_n >= s_saven) {
+	if (s_dump && s_save && s_n >= s_saven)
+	{
 		std::string s = m_dump_root + format("%05d_read_%05x_%d_%d_%d_%d_%d_%d.bmp",
-				s_n, (int)m_env.BITBLTBUF.SBP, (int)m_env.BITBLTBUF.SBW, (int)m_env.BITBLTBUF.SPSM,
-				r.left, r.top, r.right, r.bottom);
+											 s_n, (int)m_env.BITBLTBUF.SBP, (int)m_env.BITBLTBUF.SBW, (int)m_env.BITBLTBUF.SPSM,
+											 r.left, r.top, r.right, r.bottom);
```

Comment:
Dunno if you wanna un-indent this or not, just looks a bit weird like this out so far